modules/container_runtime/main.tf

```diff
@@ -65,5 +68,172 @@ resource "aws_ssm_parameter" "ssm_container_runtime_kms_key_id" {
 resource "aws_ssm_parameter" "ecs_cluster_name" {
   name  = "/${var.environment_name}/${var.container_runtime_name}/container_runtime_ecs_cluster_name"
   type  = "String"
-  value = aws_ecs_cluster.fargate_cluster.name
+  value = local.create_ecs_with_fargate == true ? aws_ecs_cluster.fargate_cluster[0].name : aws_ecs_cluster.ec2_cluster[0].name
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# ECS with EC2 instances
+# ---------------------------------------------------------------------------------------------------------------------
+# tfsec:ignore:aws-ecs-enable-container-insight
+resource "aws_ecs_cluster" "ec2_cluster" {
+  count = local.create_ecs_with_ec2 ? 1 : 0
+  name  = var.container_runtime_name
+
+  configuration {
+    execute_command_configuration {
+      logging = "NONE"
+    }
+  }
+}
+
+locals {
```

Comment:
The user_data field is only used once, so there is no need to moving it into the locals section.

Reply:
Updated.